python/pyspark/errors/error-classes.json

```diff
@@ -0,0 +1,7 @@
+{
+  "COLUMN_IN_LIST" : {
```

Comment:
Will add more error-classes in follow-up PRs.

Since this PR introduces the error class for PySpark for the first time, I made only one example for the convenience of review.